packages/publisher/electron-release-server/src/PublisherERS.ts

```diff
@@ -17,6 +17,15 @@ interface ERSVersion {
   assets: { name: string; }[];
 }
 
+type Fetch = typeof fetch;
+const fetchAndCheckStatus = async (...fetchArgs: Parameters<Fetch>): ReturnType<Fetch> => {
+  const result = await fetch(...fetchArgs);
```

Comment:
I'd prefer if the `fetch` wrapper signature was explicit:

```suggestion
const fetchAndCheckStatus = async (
  url: RequestInfo,
  init?: RequestInit,
): Promise<fetch.Response> => {
  const result = await fetch(url, init);
```